netlify.toml

```diff
@@ -16,3 +16,8 @@ renderJavaScript = false
 [functions]
 node_bundler = "esbuild"
 external_node_modules = ["@ampproject/toolbox-optimizer"]
+
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Content-Security-Policy = "script-src 'self' *.usefathom.com *.googletagmanager.com *.google-analytics.com *.netlify.com *.netlify.app *.app.netlify.com *.swiftypecdn.com *.segment.com *.segment.io plausible.io 'sha256-JcUwlbcTNdtwp5Szxufx3JIsjsem70t6DXyfcgTcAnQ=' 'sha256-MEya+PLw9XU1WNgiMij94hmgncY4GS+aQhjC2zqaAz4=' 'sha256-ETVmFY/pVNIiyKhDh3yPNf+mlhV1Z32Ax232eJ+ES7o='; style-src 'self' 'unsafe-inline' *.swiftypecdn.com; font-src 'self' fonts.gstatic.com; frame-ancestors 'none'"
```

Comment:
@katjuell we no longer use fathom and google analytics so we should not need them in the CSP anymore.

https://github.com/sourcegraph/handbook/pull/1663

Reply:
Awesome! Thanks